apps/system/utils/kdbg_date.c

```diff
@@ -153,10 +160,23 @@ static inline int date_settime(int argc, char **args)
 	long result;
 	int ret = OK;
 
+	/* Get the year */
+
+	token = args[5];
+	if (token == NULL) {
+		goto errout_bad_parm;
+	}
+
+	result = strtol(token, NULL, 10);
+	if (result < 1900 || result > 2100) {
+		goto errout_bad_parm;
+	}
+	tm.tm_year = (int)result - 1900;
```

Comment:
I don't think we should use a definition instead of all of digit.
1. 10 means decimal and all of other code also use digit.
2. I will change it as definition, TM_YEAR_BASE.
   But this is not a part of this commit. I will make a new commit to adapt it.
3. Could you ask @thapav  the reason of 2100? She made this code and I don't know why it is 2100.

Reply:
`2100` is not added or modified by @thapav. It exists from the initial commit https://github.com/Samsung/TizenRT/commit/4fc385da0f48ac9bb776ec5638506e8ad54609d2